backend/api/api.csproj

```diff
@@ -14,4 +15,8 @@
     <PackageReference Include="Swashbuckle.AspNetCore" Version="6.2.3" />
   </ItemGroup>
 
+  <ItemGroup>
+    <EditorConfigFiles Remove="C:\Users\tlne\source\repos\flotilla\backend\api\.editorconfig" />

```

Comment:
This seems like something that should not be here?

Reply:
Yes, will fix